library(tximport)
library(dplyr)

#load a list of all the tabular abundance files generated by Kallisto
my_files <- c('C:/Users/fshi38/Desktop/temp/biol6150/h5_files/mcf7_sicbx2_3_rep1.tsv',
              'C:/Users/fshi38/Desktop/temp/biol6150/h5_files/mcf7_sicbx2_3_rep2.tsv',
              'C:/Users/fshi38/Desktop/temp/biol6150/h5_files/mcf7_sicbx2_3_rep3.tsv', 
              'C:/Users/fshi38/Desktop/temp/biol6150/h5_files/mcf7_siscr_rep1.tsv', 
              'C:/Users/fshi38/Desktop/temp/biol6150/h5_files/mcf7_siscr_rep2.tsv', 
              'C:/Users/fshi38/Desktop/temp/biol6150/h5_files/mcf7_siscr_rep3.tsv')

#load names for each file
names(my_files) <- c('mcf7_sicbx2_1','mcf7_sicbx2_2','mcf7_sicbx2_3', 'mcf7_siscr_1', 'mcf7_siscr_2', 'mcf7_siscr_3')

#Using the transcript_to_gene map (https://github.com/pachterlab/kallisto-transcriptome-indices/releases)
#create a dataframe that maps each transcript to its relevant gene
test2 = read.table('./transcript_to_genes.tabular')
tx2gene = data.frame(test2)
tx2gene$V3 <- NULL

#pull the list of genes from DESeq2 that passed the tests for significance
list_of_degs <- readLines('./degs.csv')

#Use tximport command to generate new file
txi.kallisto <- tximport(my_files, tx2gene=tx2gene, type='kallisto', txOut=FALSE)

#pull only the abundance data from the tximport output and write as a csv
gene_abundances <- txi.kallisto$abundance
write.csv(gene_abundances, 'tximport_abundances_mcf7.csv')
